modules/ROOT/partials/configuration/a11ychecker_filter_issue.adoc

```diff
@@ -2,7 +2,7 @@
 
 The `a11ychecker_filter_issue` option allows Accessibility Checker issues to be removed from the Accessibility Checker report using a callback function. This option is a flexible alternative to the `a11ychecker_ignored_rules` option. This option can remove issues from the results shown in the dialog and the `getReport` API.
 
-The function will be passed the results of the <<getreport,`getReport()` API>>. To filter an issue from the Accessibility Checker report, the callback needs to return `false` for that particular issue.
+The function will be passed the results of the xref:getreport[`getReport()` API]. To filter an issue from the Accessibility Checker report, the callback needs to return `false` for that particular issue.
```

Comment:
```suggestion
The function will be passed the results of the xref:plugins/premium/a11ychecker.adoc#getreport[`getReport()` API]. To filter an issue from the Accessibility Checker report, the callback needs to return `false` for that particular issue.
```
This id doesn't appear to be on this page and instead it looks like it's meant to link back to the main A11ychecker plugin page.